src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -802,6 +802,46 @@ public String toString() {
     }
   };
 
+  public static final Builder<Map.Entry<StreamEntryID, List<StreamEntry>>> STREAM_AUTO_CLAIM
+      = new Builder<Map.Entry<StreamEntryID, List<StreamEntry>>>() {
+    @Override
+    @SuppressWarnings("unchecked")
+    public Map.Entry<StreamEntryID, List<StreamEntry>> build(Object data) {
+      if (null == data) {
+        return null;
+      }
+
+      List<ArrayList<Object>> objectList = (List<ArrayList<Object>>) data;
+      return new AbstractMap.SimpleEntry<>(new StreamEntryID(BuilderFactory.STRING.build(objectList.get(0))),
+          STREAM_ENTRY_LIST.build(objectList.get(1)));
+    }
+
+    @Override
+    public String toString() {
+      return "Map.Entry<StreamEntryID, List<StreamEntry>>";
+    }
+  };
+
+  public static final Builder<Map.Entry<StreamEntryID, List<StreamEntryID>>> STREAM_AUTO_CLAIM_ID
```

Comment:
Add a recognizable suffix with the variable name.

Reply:
Changed the variable name.